frontend/src/app/modules/theme/theme-effects.js

```diff
@@ -0,0 +1,11 @@
+import { ThemeTypes } from '../../const/theme-types'
+
+export const changeTheme = theme => {
+  const newTheme = theme === ThemeTypes.Dark ? ThemeTypes.Light : ThemeTypes.Dark
+  window.localStorage.setItem('theme', newTheme)
+
+  return {
+    theme: newTheme,
+    themeTransition: 1,
```

Comment:
The transition is reset to `1` upon every theme change - it probably doesn't need to be part of the payload, the reducer can just set it automatically as a reaction to the `themeChanged` action. Additionally, it might be better to use the `boolean` values `true` & `false` for this purpose.

Reply:
I agree, will do

Reply:
[done](https://github.com/salsita/foosball-rating/pull/81/commits/6233538904b5337673c199255c73c62943d1415e#diff-a40f089e71d55fac58c940a9aa91b1d0)